changelog.d/7829.bugfix

```diff
@@ -0,0 +1 @@
+Fix bug where we did not always pass in `app_name` or `server_name` to email templates, including e.g. for registration emails.
```

Comment:
Has this "always" been this way or was it broken at some point? (Do we need to add a "broken since v..."?)

Reply:
I *think* this has always been broken for these emails, as they were added later at different times